src/table.js

```diff
@@ -140,7 +140,10 @@ Table.createPrefixRange_ = function(start) {
  * @param {object} [options] See {@link Instance#createTable}.
  * @param {object} [options.gaxOptions]  Request configuration options, outlined
  *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
- *
+ * @param {function} [callback] The callback function.
```

Comment:
These are the response parameters: https://github.com/muraliQlogic/nodejs-bigtable/blob/9049097a93e294576dd3cf0fed639678e1fa7c9a/src/instance.js#L252-L255